content/spin/writing-apps.md

```diff
@@ -45,7 +45,7 @@ This example is the manifest for a simple HTTP application with a single compone
 
 ```toml
 # General identification information
-spin_version = "1"
+spin_manifest_version = "1"
```

Comment:
This appears elsewhere in the docs - should we rev those too?

Reply:
Covered a bunch bumping `developer/spin.toml` in https://github.com/fermyon/developer/pull/444.